samples/web/asp-web-application/Core_6/WebApplication/WebApplication.csproj

```diff
@@ -19,6 +19,7 @@
     <IISExpressUseClassicPipelineMode />
     <TargetFrameworkProfile />
     <UseGlobalApplicationHostFile />
+    <Use64BitIISExpress />
```

Comment:
Is this needed?

Reply:
Guess not? Was added by VS automatically. Reverting it means fighting with VS as it will add it back everytime you open the sample.